wallet/src/nightfall-browser/event-handlers/block-proposed.js

```diff
@@ -56,7 +56,7 @@ async function blockProposedEventHandler(data, ivks, nsks) {
     const storeCommitments = [];
     const tempTransactionStore = [];
     if (
-      (transaction.transactionType === '1' || transaction.transactionType === '2') &&
+      (Number(transaction.transactionType) === 1 || Number(transaction.transactionType) === '2') &&
```

Comment:
Number(transaction.transactionType) === '2' should be
Number(transaction.transactionType) === 2

Reply:
good catch!